show/interfaces/__init__.py

```diff
@@ -1,13 +1,30 @@
 import json
+import os
 
 import click
 import utilities_common.cli as clicommon
 import utilities_common.multi_asic as multi_asic_util
 from natsort import natsorted
 from tabulate import tabulate
 from sonic_py_common import multi_asic
+from sonic_py_common import device_info
+from swsssdk import ConfigDBConnector
+from portconfig import get_child_ports
 
 from . import portchannel
+from collections import OrderedDict
+
+HWSKU_JSON = 'hwsku.json'
+
+# Read given JSON file
+def readJsonFile(fileName):
+    try:
+        with open(fileName) as f:
+            result = json.load(f)
+    except Exception as e:
```

Comment:
>Exception [](start = 11, length = 9)

Could you catch the specific type/types? such as file exception, json format exceptions, etc?